src/core/core.helpers.js

```diff
@@ -669,6 +669,11 @@ module.exports = function(Chart) {
 	};
 	// Request animation polyfill - http://www.paulirish.com/2011/requestanimationframe-for-smart-animating/
 	helpers.requestAnimFrame = (function() {
+		if (typeof window === 'undefined') {
```

Comment:
I think this use case (server side rendering) should rely on a new platform (e.g. `platform.node`) but not sure how it would be implemented. For now it may be easier to merge these (harmless) changes if it helps projects to integrate Chart.js. 

@khorolets are these the only checks needed in order to have Chart.js running server side? What about the other specific `HTMLCanvasElement`, `CanvasRenderingContext2D`, `CanvasGradient` or even the `iframe`?